flake.nix

```diff
@@ -11,7 +11,7 @@
     ogmios.url = "github:mlabs-haskell/ogmios/c4f896bf32ad066be8edd8681ee11e4ab059be7f";
     ogmios-datum-cache = {
       url = "github:mlabs-haskell/ogmios-datum-cache";
-      flake = false;
+      flake = true;
```

Comment:
You can just delete this line, it's `true` unless otherwise specified for flake inputs